src/main/java/com/example/TracingServletContextListener.java

```diff
@@ -56,6 +53,7 @@ public void contextInitialized(ServletContextEvent servletContextEvent) {
     // TracingFilter filter = new TracingFilter(GlobalTracer.get());
     // FilterRegistration.Dynamic reg = servletContextEvent.getServletContext().addFilter("tracingFilter", filter);
     // reg.addMappingForUrlPatterns(EnumSet.allOf(DispatcherType.class), false, "/*");
+    SqlInterceptorRegistry.getInstance().register(new SqlInterceptorImpl());
```

Comment:
Maybe we should add a comment explaining this is where webapp registers the actual SqlInterceptor with the registry due to classpath reasons (this allows us to keep all the Prometheus/Tracing jars in webapp instead of having to stuff in tomcat/lib).

Reply:
I threw your comment from here above hehe